resources/webviews/aksclusterproperties/clusterproperties.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>AKS Cluster Properties</title>
+    <!-- Link to the css file -->
+    <link rel="stylesheet" href="{{cssuri}}">
+</head>
+
+<body>
+    <div class="heading">
+        <h2 class="content-header">AKS Cluster Properties of {{name}}</h2>
+    </div>
+
+    <div class="insight-container">
+        <div class="panel panel-default data-container">
+            <div class="panel-body data-container-body">
+
+                <div class="panel-body data-container-body allocatebody">
+                    <div class="panel-body">
+                        <table>
+                            <tbody>
+                                <tr>
+                                    <th>Properties</th>
+                                    <th>Values</th>
+                                </tr>
+
+                                <tr>
+                                    <td>Provisioning State</td>
+                                    <td>{{clusterData.provisioningState}}</td>
+                                </tr>
+                                <tr>
+                                    <td>FQDN</td>
+                                    <td>{{clusterData.fqdn}}</td>
+                                </tr>
+                                <tr>
+                                    <td>Kubernetes Version</td>
+                                    <td>{{clusterData.kubernetesVersion}}</td>
+                                </tr>
+                            </tbody>
+                        </table>
+                        <b>Agent Pool Properties</b>
```

Comment:
How about we move this line inside the `{{#each}}` and put the name in it, like maybe:
```
<b>Agent Pool: {{this.name}}</b>
```
Then we get a nice heading + table for each agent pool (and we're not missing out their names, which I think are needed).

I was also trying to figure out if the preferred term is 'agent pool' or 'node pool', and I can't figure it out. The docs are a complete mix...

Reply:
👍 🥷 sounds good, let me add this, mostly the property name is the actual object name is `agentProolProfiles` so might be `agent pool` is the right one for now. let me add this change, I found one of your cluster with both linux and windows node, so will be a nice one to checkout.